.github/dependabot.yml 

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Maybe a weekly interval would be sufficient

Reply:
changed

Reply:
ok